src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -280,6 +280,8 @@ public static BufferedOutputStream buffer(final OutputStream outputStream, final
      * @since 2.5
      */
     public static BufferedReader buffer(final Reader reader) {
+        // reject null early on rather than waiting for IO operation to fail
+        Objects.requireNonNull(reader, "reader");
```

Comment:
This comment is wrong and the check is not needed: The BufferedReader constructor code path already checks for null input and throws an NPE.